content/guide/core/color.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Color
+description: Create a color object with any color representation and use it in your NativeScript apps.
+---
+
+This class allows you to create a color object with all color components(alpha, red, green, blue in the [0, 255] range) from different types of color representations such as, ARGB, color name, hex, etc. The properties `ios` and `android` of an instance of this class provide the color object for the respective native platform.
```

Comment:
This class enables the creation of a color object with color components (alpha, red, green, blue) ranging from 0 to 255, using various color representations such as ARGB, color names, hex values, and more. The properties ios and android of an instance of this class provide the corresponding color object for the respective native platforms.